2020-07-19 16:08:12,481 ERROR: Exception on /new_project [GET] [in /home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/flask_login/utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "/home/jesse/PycharmProjects/Flask_template_site/app/routes.py", line 133, in new_project
    return render_template('new_project.html', title='New Project', form=form, userlist=user_list())
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/flask/templating.py", line 137, in render_template
    return _render(
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/flask/templating.py", line 120, in _render
    rv = template.render(context)
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/jinja2/environment.py", line 1090, in render
    self.environment.handle_exception()
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/jinja2/environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/jinja2/_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "/home/jesse/PycharmProjects/Flask_template_site/app/templates/new_project.html", line 2, in top-level template code
    {% import 'bootstrap/wtf.html' as wtf %}
  File "/home/jesse/PycharmProjects/Flask_template_site/app/templates/base.html", line 1, in top-level template code
    {% extends 'bootstrap/base.html' %}
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 1, in top-level template code
    {% block doc -%}
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 4, in block "doc"
    {%- block html %}
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 20, in block "html"
    {% block body -%}
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 23, in block "body"
    {% block content -%}
  File "/home/jesse/PycharmProjects/Flask_template_site/app/templates/base.html", line 49, in block "content"
    {% block app_content %}{% endblock %}
  File "/home/jesse/PycharmProjects/Flask_template_site/app/templates/new_project.html", line 12, in block "app_content"
    {{ wtf.quick_form(form) }}
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/jinja2/runtime.py", line 679, in _invoke
    rv = self._func(*arguments)
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/flask_bootstrap/templates/bootstrap/wtf.html", line 205, in template
    {{ form_field(field,
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/jinja2/runtime.py", line 679, in _invoke
    rv = self._func(*arguments)
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/flask_bootstrap/templates/bootstrap/wtf.html", line 123, in template
    {{field(class="form-control", **kwargs)|safe}}
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/wtforms/fields/core.py", line 160, in __call__
    return self.meta.render_field(self, kwargs)
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/wtforms/meta.py", line 56, in render_field
    return field.widget(field, **render_kw)
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/wtforms/widgets/core.py", line 300, in __call__
    for val, label, selected in field.iter_choices():
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/wtforms_sqlalchemy/fields.py", line 172, in iter_choices
    for pk, obj in self._get_object_list():
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/wtforms_sqlalchemy/fields.py", line 98, in _get_object_list
    query = self.query or self.query_factory()
TypeError: 'NoneType' object is not callable
2020-07-19 16:08:56,639 INFO: Flask-Template-Site startup [in /home/jesse/PycharmProjects/Flask_template_site/app/__init__.py:60]
2020-07-19 16:08:59,623 ERROR: Exception on /new_project [GET] [in /home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/flask_login/utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "/home/jesse/PycharmProjects/Flask_template_site/app/routes.py", line 133, in new_project
    return render_template('new_project.html', title='New Project', form=form, userlist=user_list())
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/flask/templating.py", line 137, in render_template
    return _render(
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/flask/templating.py", line 120, in _render
    rv = template.render(context)
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/jinja2/environment.py", line 1090, in render
    self.environment.handle_exception()
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/jinja2/environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/jinja2/_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "/home/jesse/PycharmProjects/Flask_template_site/app/templates/new_project.html", line 2, in top-level template code
    {% import 'bootstrap/wtf.html' as wtf %}
  File "/home/jesse/PycharmProjects/Flask_template_site/app/templates/base.html", line 1, in top-level template code
    {% extends 'bootstrap/base.html' %}
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 1, in top-level template code
    {% block doc -%}
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 4, in block "doc"
    {%- block html %}
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 20, in block "html"
    {% block body -%}
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 23, in block "body"
    {% block content -%}
  File "/home/jesse/PycharmProjects/Flask_template_site/app/templates/base.html", line 49, in block "content"
    {% block app_content %}{% endblock %}
  File "/home/jesse/PycharmProjects/Flask_template_site/app/templates/new_project.html", line 12, in block "app_content"
    {{ wtf.quick_form(form) }}
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/jinja2/runtime.py", line 679, in _invoke
    rv = self._func(*arguments)
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/flask_bootstrap/templates/bootstrap/wtf.html", line 205, in template
    {{ form_field(field,
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/jinja2/runtime.py", line 679, in _invoke
    rv = self._func(*arguments)
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/flask_bootstrap/templates/bootstrap/wtf.html", line 123, in template
    {{field(class="form-control", **kwargs)|safe}}
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/wtforms/fields/core.py", line 160, in __call__
    return self.meta.render_field(self, kwargs)
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/wtforms/meta.py", line 56, in render_field
    return field.widget(field, **render_kw)
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/wtforms/widgets/core.py", line 300, in __call__
    for val, label, selected in field.iter_choices():
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/wtforms_sqlalchemy/fields.py", line 172, in iter_choices
    for pk, obj in self._get_object_list():
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/wtforms_sqlalchemy/fields.py", line 98, in _get_object_list
    query = self.query or self.query_factory()
TypeError: 'BaseQuery' object is not callable
2020-07-19 16:10:42,523 INFO: Flask-Template-Site startup [in /home/jesse/PycharmProjects/Flask_template_site/app/__init__.py:60]
2020-07-19 16:11:01,000 INFO: Flask-Template-Site startup [in /home/jesse/PycharmProjects/Flask_template_site/app/__init__.py:60]
2020-07-19 16:11:06,496 ERROR: Exception on /new_project [GET] [in /home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/flask_login/utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "/home/jesse/PycharmProjects/Flask_template_site/app/routes.py", line 133, in new_project
    return render_template('new_project.html', title='New Project', form=form, userlist=user_list())
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/flask/templating.py", line 137, in render_template
    return _render(
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/flask/templating.py", line 120, in _render
    rv = template.render(context)
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/jinja2/environment.py", line 1090, in render
    self.environment.handle_exception()
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/jinja2/environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/jinja2/_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "/home/jesse/PycharmProjects/Flask_template_site/app/templates/new_project.html", line 2, in top-level template code
    {% import 'bootstrap/wtf.html' as wtf %}
  File "/home/jesse/PycharmProjects/Flask_template_site/app/templates/base.html", line 1, in top-level template code
    {% extends 'bootstrap/base.html' %}
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 1, in top-level template code
    {% block doc -%}
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 4, in block "doc"
    {%- block html %}
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 20, in block "html"
    {% block body -%}
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 23, in block "body"
    {% block content -%}
  File "/home/jesse/PycharmProjects/Flask_template_site/app/templates/base.html", line 49, in block "content"
    {% block app_content %}{% endblock %}
  File "/home/jesse/PycharmProjects/Flask_template_site/app/templates/new_project.html", line 12, in block "app_content"
    {{ wtf.quick_form(form) }}
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/jinja2/runtime.py", line 679, in _invoke
    rv = self._func(*arguments)
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/flask_bootstrap/templates/bootstrap/wtf.html", line 205, in template
    {{ form_field(field,
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/jinja2/runtime.py", line 679, in _invoke
    rv = self._func(*arguments)
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/flask_bootstrap/templates/bootstrap/wtf.html", line 123, in template
    {{field(class="form-control", **kwargs)|safe}}
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/wtforms/fields/core.py", line 160, in __call__
    return self.meta.render_field(self, kwargs)
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/wtforms/meta.py", line 56, in render_field
    return field.widget(field, **render_kw)
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/wtforms/widgets/core.py", line 300, in __call__
    for val, label, selected in field.iter_choices():
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/wtforms_sqlalchemy/fields.py", line 172, in iter_choices
    for pk, obj in self._get_object_list():
  File "/home/jesse/PycharmProjects/Flask_template_site/venv/lib/python3.8/site-packages/wtforms_sqlalchemy/fields.py", line 98, in _get_object_list
    query = self.query or self.query_factory()
TypeError: 'BaseQuery' object is not callable
2020-07-19 16:11:21,129 INFO: Flask-Template-Site startup [in /home/jesse/PycharmProjects/Flask_template_site/app/__init__.py:60]
